README.md

```
@@ -1,58 +1,312 @@
+[![SQL Server](https://img.shields.io/badge/SQL_Server-2022-%23CC2927)](https://www.microsoft.com/en-us/sql-server/sql-server-2022)
+[![Redis](https://img.shields.io/badge/Redis-%23DD0031.svg?logo=redis&logoColor=white)](https://redis.io/)
+[![JWT](https://img.shields.io/badge/JWT-Auth-%23000000?logo=jsonwebtokens)](https://learn.microsoft.com/en-us/aspnet/core/security/authentication/configure-jwt-bearer-authentication?view=aspnetcore-8.0)
+[![QRCoder](https://img.shields.io/badge/QR_Code-Generator-%23000000?logo=qrcode)](https://www.c-sharpcorner.com/article/creating-qrcode-in-asp-net-core/)
 
-| Category        | Details                                               |
-|-----------------|-------------------------------------------------------|
-| **Mentor**      | [Jaber Kibria](https://github.com/mhsjaber)           |
-| **Team Member** | [Md. Nazmul Hossain](https://github.com/nazmulhossin)     |
-|                 | [Sujan Roy](https://github.com/sujan-roy24) |
-|                 | [Yeakin Iqra ](https://github.com/yeakiniqra) |
+<h3 align="center">Fast . Convenient . Cashless</h3>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: h3

(MD033, no-inline-html)